docs/deployment/README.md#cumulus-instance-sizing

```diff
@@ -441,8 +441,8 @@ Outputs:
 
 archive_api_redirect_uri = https://abc123.execute-api.us-east-1.amazonaws.com/dev/token
 archive_api_uri = https://abc123.execute-api.us-east-1.amazonaws.com/dev/
-distribution_redirect_uri = https://abc123.execute-api.us-east-1.amazonaws.com/DEV/login
-distribution_url = https://abc123.execute-api.us-east-1.amazonaws.com/DEV/
+tea_distribution_redirect_uri = https://abc123.execute-api.us-east-1.amazonaws.com/DEV/login
+tea_distribution_url = https://abc123.execute-api.us-east-1.amazonaws.com/DEV/
```

Comment:
So these output names wouldn't change for end users unless you make a corresponding change in the template-depoy repo, which it doesn't appear that you have:

https://github.com/nasa/cumulus-template-deploy/pull/63/files

Whether we should is another question. But if end users are only deploying TEA or cumulus, it's probably not as essential

Reply:
Ah ok that makes sense. Just the outputs from `example` have changed.